addon/serializers/web-api.js

```diff
@@ -10,10 +11,10 @@ export default DS.RESTSerializer.extend({
 
     if(isCollection) {
       payload.forEach((item) => {
-        this.extractRelationships(store, payloadWithRoot, item, primaryType);
+        this.privateExtractRelationships(store, payloadWithRoot, item, primaryType);
```

Comment:
Now I'm really confused. I'll have to check to see if `extractRelationships` (or if the signature changed) was added in 1.13. If not, I'm at a loss for how the serializer is working in my project.

...I love developing to a constantly moving target. Haha.